app/models/support_interface/applications_filter.rb

```diff
@@ -117,5 +121,75 @@ def interviews_filter
         ],
       }
     end
+
+    def status_filter
+      {
+        type: :checkboxes,
+        heading: 'Status',
+        name: 'status',
+        options: [
+          {
+            value: 'unsubmitted',
+            label: 'Not submitted yet',
+            checked: applied_filters[:status]&.include?('unsubmitted'),
+          },
+          {
+            value: 'awaiting_provider_decision',
+            label: 'Awaiting provider decision',
+            checked: applied_filters[:status]&.include?('awaiting_provider_decision'),
+          },
+          {
+            value: 'interviewing',
+            label: 'Interviewing',
+            checked: applied_filters[:status]&.include?('interviewing'),
+          },
+          {
+            value: 'offer',
+            label: 'Offered',
+            checked: applied_filters[:status]&.include?('offer'),
+          },
+          {
+            value: 'pending_conditions',
+            label: 'Pending conditions',
+            checked: applied_filters[:status]&.include?('pending_conditions'),
+          },
+          {
+            value: 'conditions_met',
+            label: 'Conditions met',
+            checked: applied_filters[:status]&.include?('conditions_met'),
+          },
+          {
+            value: 'rejected',
+            label: 'Rejected',
+            checked: applied_filters[:status]&.include?('rejected'),
+          },
+          {
+            value: 'declined',
+            label: 'Declined',
+            checked: applied_filters[:status]&.include?('declined'),
+          },
+          {
+            value: 'application_withdrawn',
```

Comment:
I think this should be `offer_withdrawn`

![image](https://user-images.githubusercontent.com/42515961/128507421-ff119265-3d01-434e-9aa4-50210b8f19fc.png)